Test/Test.csproj

```diff
@@ -32,12 +32,15 @@
     <ConsolePause>false</ConsolePause>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="log4net, Version=2.0.8.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">

```

Comment:
What is this overall commit doing? Causing log4net to be installed when you install the packetdotnet nuget package? If you could clarify in the commit body that would be ideal.